.github/workflows/ruby_ci.yaml

```
@@ -1,74 +1,29 @@
 name: Ruby
+
 on:
   pull_request:
   push:
     branches:
       - master
+
 concurrency:
   group: ${{ github.head_ref }}
   cancel-in-progress: true
+
 env:
   RAILS_ENV: test
   DATABASE_URL: postgresql://postgres:@localhost/test
   DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: true
+
 jobs:
   rubocop:
     uses: theforeman/actions/.github/workflows/rubocop.yml@v0
     with:
       command: bundle exec rubocop --parallel --format github
-  test_ruby:
-    runs-on: ubuntu-latest
+
+  test:
+    name: Ruby
     needs: rubocop
-    timeout-minutes: 30
-    env:
-      BUNDLE_WITHOUT: journald:development:console:libvirt
-    services:
-      postgres:
-        image: postgres:12.1
-        ports: ['5432:5432']
-        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
-    strategy:
-      fail-fast: false
-      matrix:
-        foreman-core-branch: [develop]
-        ruby-version: [2.7]
-    steps:
-      - run: sudo apt-get update
-      - run: sudo apt-get install build-essential libcurl4-openssl-dev zlib1g-dev libpq-dev
-      - uses: actions/checkout@v2
-        with:
-          repository: theforeman/foreman
-          ref: ${{ matrix.foreman-core-branch }}
-      - uses: actions/checkout@v2
-        with:
-          path: foreman_bootdisk
-      - name: Setup Bundler
-        run: |
-          echo "gem 'foreman_bootdisk', path: './foreman_bootdisk'" > bundler.d/foreman_bootdisk.local.rb
-      - name: Setup Ruby
-        uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: ${{ matrix.ruby-version }}
-          bundler-cache: true
-      - name: Setup Node
-        if: github.event_name != 'push'
-        uses: actions/setup-node@v1
-        with:
-          node-version:  ${{ matrix.node-version }}
-      - name: Prepare test DB
-        if: github.event_name != 'push'
-        run: |
-          bundle exec rake db:create
-          bundle exec rake db:migrate
-      - name: Run plugin tests
-        if: github.event_name != 'push'
-        run: |
-          bundle exec rake test:foreman_bootdisk
-          bundle exec rake test TEST="test/unit/foreman/access_permissions_test.rb"
-      - name: 'Upload logs'
-        uses: actions/upload-artifact@v2
-        if: failure()
-        with:
-          name: logs
-          path: log/*.log
-          retention-days: 5
+    uses: theforeman/actions/.github/workflows/foreman_plugin.yml@install-npm
```

currently using a branch, should be `v0` once https://github.com/theforeman/actions/pull/18 is merged.

---

It's merged. And since I deleted the branch, this will now fail.
```suggestion
    uses: theforeman/actions/.github/workflows/foreman_plugin.yml@v0
```